pipeline/shell.py

```diff
@@ -26,6 +26,18 @@
 log = logging.getLogger(__name__)
 
 
+# ---- Some constants ----
+report_tbls = [report.SessionLevelReport,
+               report.ProbeLevelReport,
+               report.ProbeLevelPhotostimEffectReport,
+               report.UnitLevelReport,
+               report.SessionLevelCDReport,
+               report.SessionLevelProbeTrack,
+               report.ProjectLevelProbeTrack]
+
+stage = dj.config['stores']['report_store']['stage']
```

Comment:
can this be made error proof? Ideally shell should import out of the box with a minimal dj.config (db only access or unset) and fail within any 'specialized' sort of function